libretro/Makefile

```diff
@@ -304,7 +304,7 @@ else ifeq ($(platform), emscripten)
 	TARGET := $(TARGET_NAME)_libretro_emscripten.bc
 	GLES := 1
 	CPUFLAGS +=
-	PLATCFLAGS += -msimd128 -msse3 -mssse3 -msse4.1 -pthread -D__EMSCRIPTEN__ -DUSE_ASAN
+	PLATCFLAGS += -msimd128 -msse3 -mssse3 -msse4.1 -pthread -D__EMSCRIPTEN__
```

Comment:
when doing emscripten, surely sse* flags aren't relevant?

Reply:
They are, actually. I just tested, making sure they were. When removing them, it requires that you have at least `-msimd128` set, which basically implies all the other sse flags.

Reply:
EDIT: compiling with only `-msimd128` produces this error:

![Screenshot 2023-07-31 175532](https://github.com/hrydgard/ppsspp/assets/77750390/502ff7fc-9742-43f3-a429-664f87140f74)